php/libtofs.jinja

```diff
@@ -0,0 +1,114 @@
+{%- macro files_switch(source_files,
```

Comment:
Get the latest version of this file from the `template-formula`.

Reply:
done